axelrod/tests/unit/test_strategy_transformers.py

```diff
@@ -1001,20 +1001,6 @@ class TestFlippedDualTransformer(TestPlayer):
     }
 
 
-class TestIdentityDualTransformer(TestPlayer):
```

Comment:
Why remove this?

Reply:
Repetition; an identical class was defined earlier..

Reply:
Great :+1: Thank you.